src/components/viewsComponents/dashboard/elements/ClosestEvents.tsx

```diff
@@ -1,85 +1,42 @@
-import { useMemo } from 'react';
 import { useNavigate } from 'react-router-dom';
 import EventIcon from '@mui/icons-material/Event';
 import { Typography, List, ListItem, Paper, Box } from '@mui/material';
 import { staticEvents } from 'components/viewsComponents/events/constants/EventsData';
+import '../styles/ElementsStyles.scss';
+import { format } from 'date-fns';
+import { getClosestEvents } from '../utils/EventsHelpers';
 
 const ClosestEvents = () => {
-  const now = new Date();
+  const closestEvents = getClosestEvents(staticEvents);
```

Comment:
Make it on useMemo

Reply:
used useMemo to memoize staticEvents computation.